### PR TITLE
Move to GitHub auth

### PR DESCRIPTION
**Pull request by jessieay**
* Using github team member auth: https://github.com/jonmagic/omniauth-github-team-member
* Only members of 18F team can log in
* Closes https://github.com/18F/dolores-landingham-bot/issues/181

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -112,58 +129,69 @@ you can contact Jessie Young. Otherwise, you can create a Slack bot
...
-  if is_permitted?(auth_email)
-
-  # Method
-  def is_permitted?(auth_email)
-    /#{ENV['AUTH_DOMAIN']}/.match(auth_email)
+  # app/controllers/auth_controller
+
+  def oauth_callback
+    # if team_member?
+      dev_test_email = your_email@example.com
```
Comment (yozlet) [inline CONTRIBUTING.md]: I realise that most people are going to know the right thing to do here, but how about putting that email address in quotes?

File: db/schema.rb [hunk 2]
```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```
Comment (yozlet) [inline db/schema.rb]: What caused these `schema.rb` changes?

**Comment by yozlet** [comment]
Added a couple of minor comments but neither of these is a blocker, just something to look at before you :shipit:. In other words, 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (jessieay) [inline CONTRIBUTING.md]: hah goooood idea

File: db/schema.rb (hunk 2 above)
Comment (jessieay) [inline db/schema.rb]: I think I forgot to run migrations after updating to Rails 5 earlier this week :(

**Pull request merged**